essim/src/ssim_ssim.c

```diff
@@ -19,6 +19,8 @@
 
 #define max(a, b) (((a) > (b)) ? (a) : (b))
 
+uint32_t div_lookup[65536];
+
 static uint32_t isqrt(const uint64_t v) {
```

Comment:
I think this might predate the changes here, but this is unused and throws a warning, let's clean this up while we're at it?